md-rest-api/src/main/java/com/wipro/ats/bdre/md/rest/UserRolesAPI.java

```diff
@@ -181,4 +191,27 @@ public Object execute(String[] params) {
     }
 
 
+    @RequestMapping(value = {"/options", "/options/"}, method = RequestMethod.POST)
+    @ResponseBody public
+    RestWrapperOptions listOptions(Principal principal) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the unused method parameter(s) "principal". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1172)